src/node_os.cc

```diff
@@ -121,14 +121,15 @@ static void GetCPUInfo(const FunctionCallbackInfo<Value>& args) {
   // [model, speed, (5 entries of cpu_times), model2, speed2, ...]
   std::vector<Local<Value>> result(count * 7);
   for (int i = 0; i < count; i++) {
+    int j = i * 7;
```

Comment:
Instead of this, `int j = 0` could be added to the loop initialization and the last index assignment changed to `j++`.

Reply:
@mscdex - followed your suggestions, ptal. thanks!